keyboards/kinesis/keymaps/farmergreg/rules.mk

```
@@ -0,0 +1,12 @@
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
+
+KEY_OVERRIDE_ENABLE = yes		# https://docs.qmk.fm/#/feature_key_overrides
+TAP_DANCE_ENABLE = yes			# https://docs.qmk.fm/#/feature_tap_dance
```

This can be condensed down to only the rules that differ from the target board.
```suggestion
BOOTMAGIC_ENABLE = no
COMMAND_ENABLE = yes
NKRO_ENABLE = yes
KEY_OVERRIDE_ENABLE = yes
TAP_DANCE_ENABLE = yes
```

---

updated as suggested, thanks!